src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -934,6 +934,15 @@ public String getTeamNameForEmail(String email) {
             return Sanitizer.sanitizeForHtml(teamName);
         }
     }
+    
+    public String getUnsanitizedTeamNameForEmail(String email) {
+        String teamName = emailTeamNameTable.get(email);
+        if (teamName == null || email.equals(Const.GENERAL_QUESTION)) {
+            return Const.USER_NOBODY_TEXT;
+        } else {
+            return teamName;
+        }
+    }
```

Comment:
Ah, yes. This is a temporary method that should be removed when a more systematic cleanup is done. The original method was used in other places, so I didn't want to affect behaviour anywhere else yet. I'll add a TODO here.